docs/components/buttongroup/elements.md

```diff
@@ -1 +0,0 @@
-This component is only supported in Warp Vue at the moment.
```

Comment:
This import should be removed from `/buttongroup/index.md`. Maybe other pages also need double-checking that there are no missing imports errors 😊

Reply:
wops! yes, I've gone through everything and it should work now 🙂